lib/legacy/class.html

```diff
@@ -166,28 +248,57 @@
         /**
          * @return {void}
          */
+        // Called on element prototype
         _registered() {
-          super._registered();
           /* NOTE: `beforeRegister` is called here for bc, but the behavior
            is different than in 1.x. In 1.0, the method was called *after*
            mixing prototypes together but *before* processing of meta-objects.
            However, dynamic effects can still be set here and can be done either
            in `beforeRegister` or `registered`. It is no longer possible to set
            `is` in `beforeRegister` as you could in 1.x.
           */
+          const proto = this;
+          if (proto.hasOwnProperty('__behaviors')) {
```

Comment:
This is just what's in the closure, i.e.
`if (behaviors) {`

Reply:
Done.